graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -543,17 +558,12 @@ def _rename_or_suppress_or_ignore_name_and_add_to_record(
             new_name_string in self.reverse_name_map
             and self.reverse_name_map[new_name_string] != name_string
         ):
-            raise SchemaNameConflictError(
-                '"{}" and "{}" are both renamed to "{}"'.format(
-                    name_string, self.reverse_name_map[new_name_string], new_name_string
-                )
-            )
+            # At least two different types are mapped to the same new_name_string
+            if new_name_string not in self.name_conflicts:
+                self.name_conflicts[new_name_string] = {self.reverse_name_map[new_name_string]}
+            self.name_conflicts[new_name_string].add(name_string)
```

Comment:
I think this could be streamlined with `setdefault` https://www.geeksforgeeks.org/python-dictionary-setdefault-method/

Reply:
I think this came up at one point and neither Predrag or I had super strong opinions about it-- I find the existing code slightly easier to read, but that's just my personal preference

Reply:
I find `setdefault()` slightly preferable but I wasn't going to block your PR over it 😄